_deploy/r/gnoswap/common/tick_math.gno

```diff
@@ -43,34 +43,40 @@ var binaryLogConsts = [8]*u256.Uint{
 
 var (
 	shift1By32Left = u256.MustFromDecimal("4294967296") // (1 << 32)
+	maxTick        = int32(887272)
 )
 
 func TickMathGetSqrtRatioAtTick(tick int32) *u256.Uint { // uint160 sqrtPriceX96
 	absTick := abs(tick)
-	if absTick > 887272 { // MAX_TICK
+	if absTick > maxTick { // MAX_TICK
```

Comment:
drop comments, no need since var name is maxTick

Reply:
@r3v4s https://github.com/gnoswap-labs/gnoswap/pull/421/commits/906ab514c514565999bc48dc139a8a847b3e4422